include/class.config.php

```diff
@@ -673,19 +673,20 @@ function alertAcctManagerONNewMessage() {
         return ($this->get('message_alert_acct_manager'));
     }
 
-    function alertONNewNote() {
+    //TODO: change note_alert to activity_alert
```

Comment:
I added the `db-migration` label to signify renaming the config settings on some upgrade task